CMakeLists.txt

```diff
@@ -27,6 +27,8 @@ add_custom_target(all_ice40_tests)
 set(ENV_DIR ${symbiflow-arch-defs_SOURCE_DIR}/env/conda/envs/symbiflow_arch_def_base/ CACHE PATH "Path to environment")
 set(YOSYS_DATADIR ${ENV_DIR}/share/yosys CACHE PATH "Path to yosys data directory")
 set(VPR_CAPNP_SCHEMA_DIR ${ENV_DIR}/capnp CACHE PATH "Path to VPR schema directory")
+set(INSTALL_DEVICE "small_devices" CACHE STRING "The device or device set to install")
```

Comment:
I assume you meant for one of those `device` to be `devices`?  It's probably worth adding a comment that `xc7a200t` is specifically not included in "small_devices"?

Reply:
I meant "device set", but maybe "set of devices" is clearer? I thought about changing `INSTALL_DEVICE` to `INSTALL_DEVICE_SET`, but `small_devices` is the only device set and will probably be removed at some point.

Reply:
I changed to the comment to "The device to install, or 'small_devices', which includes all devices except xc7a200t."